src/i18n/translations.ts

```diff
@@ -188,9 +188,11 @@ export default {
         permissionSelectExtra: 'Wählen Sie die Berechtigung aus, die die Nutzer erhalten sollen.'
       },
       ApplicationInfoModal: {
-        clientVersion: 'Admin-Version',
-        backendVersion: 'SHOGun-Version',
+        clientAbout: 'Über',
+        clientVersion: 'Admin Version',
+        backendVersion: 'Backend Version',
```

Comment:
I think should be either `Admin-Version` or `Adminversion` and `Backend-Version` / `Backendversion` respectively